avd_docs/aws/autoscaling/AVD-AWS-0341/docs.md

```diff
@@ -0,0 +1,13 @@
+
+Ensures all Auto Scaling groups have ELB health check active..
```

Comment:
```suggestion
Ensures all Auto Scaling groups have ELB health check active.
```

Reply:
You'll probably need to fix this in the rule itself and not in the generated markdown.

Reply:
resolved